test/plugin/test_compressable.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../helper'
+require 'fluent/plugin/compressable'
+
+class CompressableTest < Test::Unit::TestCase
+  include Fluent::Plugin::Compressable
+
+  sub_test_case '#compress' do
+    test 'compress data' do
+      str = 'text data'
+      compressed_str = compress(str)
+      assert_not_equal compressed_str, str
+    end
```

Comment:
Please use `assert_equal` with fixed compressed data for specified (fixed) data source.
And, use a bit more longer data source :)